wcs-functions.php

```diff
@@ -30,16 +30,18 @@
 
 /**
  * Check if a given object is a WC_Subscription (or child class of WC_Subscription), or if a given ID
- * belongs to a post with the subscription post type ('shop_subscription')
+ * belongs to a post with the subscription post type ('shop_subscription').
  *
  * @since  1.0.0 - Migrated from WooCommerce Subscriptions v2.0
+ *
+ * @param mixed $subscription A WC_Subscription object or post ID
```

Comment:
Again, here so that we are not fixated to post only. Feel free to modify.

```suggestion
 * @param mixed $subscription A WC_Subscription object or an ID.
```